src/components/atoms/IconInput/IconInput.scss

```diff
@@ -0,0 +1,41 @@
+@import "scss/constants";
+
+.icon-input {
+  position: relative;
+
+  &__input {
+    height: 100%;
+
+    &--icon-from-left {
+      padding-left: 2.5rem;
+    }
+
+    &--icon-from-right {
+      padding-right: 2.5rem;
+    }
+
+    &:focus + .icon-input__icon {
+      opacity: 0.6;
+    }
+  }
+
+  &__icon {
+    display: flex;
+    position: absolute;
+    top: 0;
+    height: 100%;
+    width: 40px;
```

Comment:
How will this work with responsive sizing? Do we always want this to be `40px`?

Reply:
a good question.
I think we need @mathieud here. I can show how it looks like right now.  There are 2 input fields where I'm using my `InputField` component: 

![image](https://user-images.githubusercontent.com/63313732/114013569-5aad5480-9870-11eb-9bfd-10252ee13f42.png)

the width of the icon (as well as the value of input padding ) is 2.5rem for both cases

In my opinion, if there is another size needed you can always add your custom class to `InputField` component and assign the width you need for a particular input.

What do you think? Is that okay with you?